### PR TITLE
Update to rxdart: ^0.27.0

### PR DESCRIPTION
**Pull request by petoknm**
and fix some null-related warnings while we're at it.

### CODE REVIEW

File: ios/flutter_blue.podspec [hunk 1]
```diff
@@ -23,7 +23,7 @@ Bluetooth Low Energy plugin for Flutter.
     ss.source_files = "gen/*.pbobjc.{h,m}", "gen/**/*.pbobjc.{h,m}"
     ss.header_mappings_dir = "gen"
     ss.requires_arc = false
-    ss.dependency "Protobuf", '~> 3.11.4'
+    ss.dependency "Protobuf", '~> 3.11'
   end
```
Comment (MrCsabaToth) [inline ios/flutter_blue.podspec:27]: Did you downgrade Protobuf version?

File: ios/flutter_blue.podspec [hunk 2]
```diff
@@ -23,7 +23,7 @@ Bluetooth Low Energy plugin for Flutter.
     ss.source_files = "gen/*.pbobjc.{h,m}", "gen/**/*.pbobjc.{h,m}"
     ss.header_mappings_dir = "gen"
     ss.requires_arc = false
-    ss.dependency "Protobuf", '~> 3.11.4'
```
Comment (MrCsabaToth) [inline ios/flutter_blue.podspec:26]: Did the generated protobuf file change with this version change?

Comment (petoknm) [reply ios/flutter_blue.podspec:26]: Sorry, I didn't mean to include the last two commits (protobuf related) in this PR. Should I delete them, or have you already cherry picked them onto your [fork](https://github.com/CsabaConsulting/flutter_blue/commit/159f6f4cfd0d145daf642b0927bd3856164b68ce)?

Comment (MrCsabaToth) [reply ios/flutter_blue.podspec:26]: I'm good, no worries.

Comment (MrCsabaToth) [reply ios/flutter_blue.podspec:26]: I like your PR BTW. I saw that particular version change in other PRs

Comment (MrCsabaToth) [reply ios/flutter_blue.podspec:26]: I have an own fork, which has multiple PRs merged into it. Pretty sad how stale is flutter_blue.

**Comment by petoknm** [comment]
Closing in favor of #880

**Pull request closed without merging**